app/windows/Settings/Components/AppPanel.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import Settings from 'electron-settings'
+import { observer } from 'mobx-react'
+
+import { Pane, CheckBox } from 'react-photonkit'
+
+/**
+ * Repository Panel
+ */
+@observer
+class AppPanel extends React.Component {
```

Comment:
Do we really need a new panel for one option? Can we add it to the Daemon settings?

Reply:
I've added it to the connectivity panel, we don't have a daemon panel anymore